### PR TITLE
Adds an 'exposed' config option, closes #105

### LINKED ISSUE #105: For packages, take the "tests" config from "exposed-modules" in elm.json
One issue with using this tool over the long run is that you forget you need to update the `elm-verify-examples.json` file with every module you add (this totally happened to me on elm-visualization - I'm dreading how many broken examples I'll find 😢).

However, for packages we already have a list of documented modules - the `"exposed-modules"` option in elm.json. Of course there may be cases where you for some reason don't want `elm-verify-examples` to run on some module, so supporting a manual configuration mode makes sense. But for packages I think this would be a more sensible default preventing this sort of user error.

### PR DESCRIPTION
**Pull request by gampleman**
Closes #105.

### CODE REVIEW

File: bin/cli-helpers.js [hunk 1]
```diff
@@ -13,16 +13,57 @@ function loadVerifyExamplesConfig(configPath) {
...
+    try {
+      elmJson = require(elmJsonPath);
+    } catch (e) {
+      console.error(
+        "Config asks for 'exposed', but could not find elm.json at " +
+          elmJsonPath
+      );
+      process.exit(1);
+    }
+    if (elmJson.type == "package" && config.tests === "exposed") {
```
Comment (lue-bird) [inline bin/cli-helpers.js:44]: Don't we already know here that `config.tests === "exposed"` is true here?

**Review by lue-bird — COMMENTED** [review]
As far as I can see, the readme is not included.
Should we include it (and mention that)? And if not, it would be nice to have `exposed` as an element among others:
```json
{ "tests" : [ "exposed", "InternalModule", , "Util", "./README.md" ] }
```

File: bin/cli-helpers.js (hunk 1 above)
Comment (gampleman) [reply bin/cli-helpers.js:44]: Oh yeah, I was gonna add another feature but changed my mind there.

**Review by stoeffel — APPROVED** [review]
looking great! thx for

**Pull request merged**